From 33b70718df93d99864043ba197a95267d12354d6 Mon Sep 17 00:00:00 2001
From: Amit Khatri <amit.khatri@mediatek.com>
Date: Thu, 6 Apr 2023 21:37:33 +0800
Subject: [PATCH 26/87] mtk: mac80211: inrease beacon loss count

as per mt7996 code beacone loss time out is
4 seconds.
in 2G connection getting beacon loss logs in routed client
scenario.

so increasing beacon loss count from 7 to 20

Signed-off-by: Amit Khatri <amit.khatri@mediatek.com>
---
 net/mac80211/mlme.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/mac80211/mlme.c b/net/mac80211/mlme.c
index f9786b7a..f3414858 100644
--- a/net/mac80211/mlme.c
+++ b/net/mac80211/mlme.c
@@ -68,7 +68,7 @@ MODULE_PARM_DESC(max_probe_tries,
  * probe on beacon miss before declaring the connection lost
  * default to what we want.
  */
-static int beacon_loss_count = 7;
+static int beacon_loss_count = 20;
 module_param(beacon_loss_count, int, 0644);
 MODULE_PARM_DESC(beacon_loss_count,
 		 "Number of beacon intervals before we decide beacon was lost.");
-- 
2.45.2

